demo/Makefile

```diff
@@ -0,0 +1,77 @@
+
+# Domain to use when starting flynn
+CLUSTER_DOMAIN := demo.localflynn.com
+CLUSTER := default
+BOOTSTRAP_LOG_FILE := bootstrap.${CLUSTER}.log
+
+# Providers: virtualbox, vmware_fusion, aws
+PROVIDER := virtualbox
+
+
+.PHONY: install-cli init bootstrap up down halt update ssh reload token destroy dashboard ssh-cmd
+
+
+# Install flynn cli on host
+install-cli:
+	L=/usr/local/bin/flynn && curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >$L && chmod +x $L
+
+# Initialize a new cluster
+init:
+	@$(MAKE) up
+	@$(MAKE) bootstrap
+	flynn key add
+
+# Bootstrap the cluster
+# This should only be called once per cluster
+# Runs bootstrap in VM and cluster add on host
+bootstrap:
+	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee ${BOOTSTRAP_LOG_FILE}
+	(grep '^flynn cluster add .*$$' ${BOOTSTRAP_LOG_FILE} || echo exit 1) | sed "s/$$(printf '\r')\$$//" | bash
+	rm ${BOOTSTRAP_LOG_FILE}
+
+# Start the VM and flynn
+up:
+	CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" vagrant up --provider ${PROVIDER}
+	@$(MAKE) CMD='sudo start flynn-host || echo flynn-host already running or unable to start' ssh-cmd
+
+# Stop flynn
+down:
+	@$(MAKE) CMD='sudo stop flynn-host' ssh-cmd
+
+# Halt the VM
+halt:
+	vagrant halt
+
+# Update the VM image to the lastest flynn vagrant box
+update:
+	vagrant box update
+
+# SSH into VM
+ssh:
+	vagrant ssh
+
+# Stop, start, and reprovision VM
+# Run to fix configuration or provisioning problems
+reload:
+	vagrant reload --provision
```

Comment:
This doesn't redo any provisioning, could call it `reboot` or change it to `destroy` + `init`.